frontend/src/metabase/components/Logs.jsx

```diff
@@ -11,8 +11,8 @@ import "react-ansi-style/inject-css";
 import _ from "underscore";
 
 export default class Logs extends Component {
-    constructor(props, context) {
-        super(props, context);
+    constructor() {
```

Comment:
Is leaving off `props` ok here? I suppose it's probably fine since we don't use props, just not sure if it's a good habit to get in. Have you seen any best practices around that?